modules/accessing-the-odh-dashboard.adoc

```diff
@@ -0,0 +1,24 @@
+:_module-type: PROCEDURE
+//pv2hash: 
+
+[id='accessing-the-odh-dashboard_{context}']
+= Accessing the Open Data Hub dashboard
+
+[role='_abstract']
+You can access and share the URL for your Open Data Hub dashboard with users to let them log in and work on their data models.
+
+.Prerequisites
+* You have installed the Open Data Hub Operator.
+
+.Procedure
+. In the OpenShift web console, select *Networking* -> *Routes*.
+. Click *Project* and select the *odh* project.
```

Comment:
Same as previous comment.

Reply:
Here's a related tip/clarification, from the IBM style guide:
https://www.ibm.com/docs/en/ibm-style?topic=elements-ui
Under Technical Elements> UI Elements
----
In most cases, avoid describing the result of an action. If you think that the context is clear, do not provide locational information. If the context isn't clear, and you must orient the user to the location where that step is performed, provide locational information at the start of the sentence.

Incorrect example 

   1. Click Fonts. The Fonts list is displayed.
    2. Select Times New Roman.

Correct example If the context is clear

   1.  Click Fonts.
   2.  Select Times New Roman.

Correct example If the context isn't clear

    1. Click Fonts.
    2. From the Fonts list, select Times New Roman.

Reply:
Thanks a lot for adding that, Melissa. In some cases here, I _do_ think the context/location isn't necessarily clear and would benefit from further clarification. But the advice on putting it at the start of the following action is new to me - it is part of the most recent update for the IBM style guide or am I using a convention from Microsoft? 

Either way, I actually really disliked the convention of appending the result to the end of the previous step - it made for really stuffy, old-fashioned doc. So I'm glad we're moving on from that :)

Apologies, Amy  - feel free to throw out any of my comments that you feel don't apply, based on what Melissa included here.

Reply:
It's a good discussion - I'm not sure when it was added to the style guide but I also prefer it. If you had not commented on this, I would not have looked it up :)

Reply:
How is this?
. In the OpenShift web console, select *Networking* -> *Routes*.
. From the *Routes* page, click the *Project* list and select the `odh` project. The list of routes filters to display only routes in the `odh` project.